docs/manual/forms/examples/prefill.rst

```diff
@@ -0,0 +1,48 @@
+=======
+Prefill
+=======
+
+In dit voorbeeld tonen we hoe u een prefill kunnen gebruiken.  Prefill is een optie zodat
+een formulier prefilled met informatie is.  Met dit voorbeeld tonen we hoe u de
```

Comment:
Is there a dutch word for `prefilled`?

Reply:
I use prefilled, but technically it's "voorinvullen"